src/main/java/com/grud/DevelopersGrud.java

```diff
@@ -0,0 +1,75 @@
+package com.grud;
+
+import lombok.SneakyThrows;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DevelopersGrud extends Grud {
+    private static final String ADD_NEW_DEVELOPER_SQL_QUERY = "INSERT INTO `developers` VALUES(NULL,? ,? ,? ,? );";
+    private static final String READ_ALL_DATA_SQL_QUERY = "SELECT * FROM `developers`;";
+    private static final String DELETE_BY_ID_SQL_QUERY = "DELETE FROM `developers` WHERE `id` = ?;";
+    private static final String CHANGE_SALARY_DEVELOPER_SQL_QUERY = "UPDATE `developers` SET `salary` = ? WHERE `name` = ?;";
+
+    @SneakyThrows
+    public int createData(String name, int age, boolean sex, int salary) {
+        Connection connection = ConnectionDb.getConnection();
+        PreparedStatement preparedStatement = connection.prepareStatement(ADD_NEW_DEVELOPER_SQL_QUERY);
+        preparedStatement.setString(1, name);
+        preparedStatement.setInt(2, age);
+        preparedStatement.setBoolean(3, sex);
+        preparedStatement.setInt(4, salary);
+        int rowInserted = preparedStatement.executeUpdate();
+        close(preparedStatement);
+        close(connection);
+        return rowInserted;
+    }
+
+    @SneakyThrows
+    @Override
+    public List readAllData() {
```

Comment:
Why not return a list of developers? 
`public List readAllData()  -> public List<Developer> findAll() or findAllDevelopers()`